components.txt

```diff
@@ -6,6 +6,7 @@
 - [Trixieapp/react-virgin]()
 - [ayoubdev/react-native-android-kit]()
 - [wix/react-native-ui-lib]()
+- [react-native-training/react-native-elements](https://github.com/react-native-training/react-native-elements)
```

Comment:
The url in the () is the image/gif to be used in the listing. 
If provided, this image will be used instead of the one fetched through crawling.